lib/vhost/vhost_user.c

```diff
@@ -2131,6 +2168,11 @@ vhost_user_get_vring_base(struct virtio_net **pdev,
 	/* We have to stop the queue (virtio) if it is running. */
 	vhost_destroy_device_notify(dev);
 
+	/* Device is not running after get vring base, we should unmask driver_ok.
```

Comment:
all of these changes for get and set config etc, has nothing to do with blk.

split the patches to two.
one for blk actual work and one for this.